app/task_routes.py

```diff
@@ -0,0 +1,108 @@
+from flask.json import jsonify
+from app import db
+from app.models.task import Task
+from flask import Blueprint, request
+from sqlalchemy import desc
+from datetime import datetime
+import os
+from dotenv import load_dotenv
+import requests
+
+load_dotenv()
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+@tasks_bp.route("", methods=["POST"], strict_slashes=False)
+def create_new_task():
+    new_task_data = request.get_json()
+    if new_task_data.keys() >= {"title", "description", "completed_at"}:
+        new_task = Task(**new_task_data)
+        db.session.add(new_task)
+        db.session.commit()
+        return {"task": new_task.task_to_json()}, 201
```

Comment:
Flask will handle returning a dictionary gracefully, but I recommend using either jsonify or make_response to add an extra layer of error checking.